Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:2.7
```

Comment:
Did you mean to add this file in? There's nothing 'bout it in the PR description.

Reply:
I doubted if opening a new PR would be better but ended up going with all in. I can split it if it makes more sense.

Reply:
I've opened another PR for docker #69